### DISCUSSION TITLE
Change phi1 range of ODF diagram from 0-360 to 0-90

### DISCUSSION THREAD

**Discussion by JinkuiMeng** [mtex-toolbox/mtex#820] [Ask Anything]
For HCP or FCC metal, how to realize the phi1 range of its ODF diagram is 0-90 degrees, is it carried out by defining their symmetry? Do you have any reference cases on the official website?

That is
From：
![图片](https://user-images.githubusercontent.com/64880536/109252421-7b719b80-7828-11eb-89ce-aaaf90ea9e83.png)

to:
![图片](https://user-images.githubusercontent.com/64880536/109252464-917f5c00-7828-11eb-85e8-627dbe3046c5.png)

**Comment by kilir** [comment]
Hi,
why would you want to discard this information or do you have specimen symmetry? Have a look here: https://mtex-toolbox.github.io/EulerAngleSections.html

Cheers,
Rüdiger

**Reply by JinkuiMeng** [reply]
Thank you for reply! Case I think it's a orthotropic specimen. It actually comes from the plate obtained by rolling. Is the symmetry of annealed and cold rolled sheet considered to be orthotropic?

**Reply by JinkuiMeng** [reply]
When the sample symmetry is defined, the 90-270 part is directly discarded, and the 0-90 part is exactly the same as the previous one.

`odf.SS = specimenSymmetry('222');`

best,

Jinkui

**Reply by kilir** [reply]
Hi Jinkui,
see this answer as well https://github.com/mtex-toolbox/mtex/discussions/770#discussioncomment-363917

Cheers,
Rüdiger

**Reply by JinkuiMeng** [reply]
Thanks a lot!
